test/TestProjects/SubClients-LowLevel/SubClients-LowLevel.json

```diff
@@ -0,0 +1,63 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-19",
+    "title": "LlcSubClients"
+  },
+  "host": "localhost:3000",
+  "schemes": [
+    "http"
+  ],
+  "produces": [ "application/json" ],
+  "consumes": [ "application/json" ],
+  "paths": {
+    "/root/{cachedParameter}": {
```

Comment:
What case is this test testing?

Reply:
It validates that we avoid naming collision of the fields. Parent client has client parameter `cachedParameter` and `GetParameterClient` method that caches the result. Both would try to create `_cachedParameter` field.

Reply:
Is there a way we could either rename files or add comments to make this clear?  I would not have guessed this from the test files.

Reply:
Comments to the json input?

Reply:
Or rename of SubClients-LowLevel to indicate a more specific case being tested?

Reply:
No, the project itself tests subclients in general. This particular parameter though tests resolution of field names collision.